articles/event-hubs/event-hubs-faq.yml

```diff
@@ -194,6 +194,11 @@ sections:
         answer: |
           [!INCLUDE [event-hubs-partition-count](./includes/event-hubs-partition-count.md)]
           
+      - question: |  
+          Can partition count be increased in Standard tier of Event Hub?
+        answer: |
+          No, it is not possible as Partitions are immututable in Standard tier. Dynamic additions of partitions is available only in premium and dedicated tiers of Event Hubs.
```

Comment:
```suggestion
          No, it's not possible because partitions are immutable in the Standard tier. Dynamic addition of partitions is available only in premium and dedicated tiers of Event Hubs.
```